bitnami/energy-platform/kafka-bitnami/upstream/kustomization.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - kafka.yaml
+
+patches:
+  # Remove "app.kubernetes.io/instance" and "helm.sh/chart" labels as well as
+  # reference to these labels in selectors and affinity fields.
+  #
+  # Bitnami puts version string there and on Chart updates it means that
+  # Service labels are updated, but StatefulSet labels are not (immutable), so:
+  #   1. We are unable to "kubectl apply StatefulSet changes"
+  #   2. Service has no endpoints because the selector
+  #
+  # Remove these labels so we can cleanly update Chart versions
```

Comment:
Should this be included?

Reply:
looks like comments to me to explain why we are doing the lines below with patch remove in kustomization. As this is all generated code based on the target in makefile and the templates used, I think probably best to keep it the same, as if ever regenerated the manual changes we do will be overwritten